store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -501,6 +502,17 @@ public CarbonLoadModel buildLoadModel(Schema carbonSchema)
     }
     // for the longstring field, change the datatype from string to varchar
     this.schema = updateSchemaFields(carbonSchema, longStringColumns);
+    if (sortColumns != null && sortColumns.length != 0) {
+      if (options == null || options.get("sort_scope") == null) {
+        // If sort_columns are specified and sort_scope is not specified,
+        // change sort scope to local_sort as now by default sort scope is no_sort.
+        if (options == null) {
+          options = new HashMap<>();
+        }
+        //TODO: add in carbon property instead of load options
```

Comment:
It should be tableproperty not carbon property

Reply:
there are test cases, that just set sort scope in carbon property but not sort columns, that test case fails as table property that I set has higher priority than carbon properties

Reply:
example, sort_columns are mentioned in table properties, but sort_scope is present as "Batch_sort" in carbon properties, so in this time If I set in table properties, test case fails. 

that too they set carbon properties after creating the table !

Reply:
ok. changed back to table properties